src/app/components/hris/employees/new-employee/new-employee.component.html

```diff
@@ -100,11 +100,11 @@ <h1 class=" mb-0 mb-md-1" id="heading">Employee Details</h1>
                     (input)="onCellphoneInputCheck(inputField.value ?? '')"
                   ></ngx-mat-intl-tel-input>
                   <mat-error *ngIf="newEmployeeForm.controls['cellphoneNo'].hasError('required') && newEmployeeForm.controls['cellphoneNo'].value=='' ">
-                    Phone Number Is Required
+                    Cellphone Number Is Required
                   </mat-error>
                   <mat-error
                     *ngIf="newEmployeeForm.controls['cellphoneNo'].value != '' && newEmployeeForm.controls['cellphoneNo'].invalid">
-                    This Is Not a Valid Phone Number
+                    This is not a valid Cellphone number.
```

Comment:
![image](https://github.com/user-attachments/assets/38ab0499-001c-41cc-b765-be988eb39743)

Reply:
checking with wouter

Reply:
This is not a valid cellphone number.  this is what wouter wants